openvdb/math/Math.h

```diff
@@ -62,6 +62,8 @@ template<> inline std::string zeroVal<std::string>() { return ""; }
 /// Return the @c bool value that corresponds to zero.
 template<> inline bool zeroVal<bool>() { return false; }
 
+namespace math {
+
```

Comment:
Important: This moves the below `inline std::string operator+` ops into the math namespace so ADL picks them up from cwiseAdd